book/src/installers/homebrew.md

```diff
@@ -2,13 +2,34 @@
 
 > since 0.2.0
 
-This provides a [Homebrew](https://brew.sh) formula which allows users to `brew install` your package. Since it installs to a location on the user's `PATH`, it provides a simple and convenient installation method for users who already have Homebrew available. When published to a [tap](https://docs.brew.sh/Taps) (package repository), this gives your users an easy way to both install your package and to keep it up to date using `brew update` and `brew upgrade`. It fetches the same prebuilt macOS binaries as the shell installer.
+cargo-dist can automatically build and publish [Homebrew](https://brew.sh) formulae (packages) for your application. Users can install your application with an expression like `brew install axodotdev/tap/axolotlsay` and automatically get updates whenever they update their Homebrew packages. The homebrew package will fetch your prebuilt [archives](../artifacts/archives.md), and install them in the traditional homebrew directory structure.
 
-cargo-dist can, optionally, publish your formula to a tap repository for you on every release. To enable this, add a `tap` field to your `Cargo.toml` pointing to a GitHub repository that you control and add `homebrew` to the `publish-jobs` field. The repository name must start with `homebrew-`. For example:
+*Building* a formula is pretty straight-forward, but publishing it requires you to create a your own [Homebrew tap](https://docs.brew.sh/Taps) (package repository), because [the core homebrew tap](https://github.com/Homebrew/homebrew-core) does not accept prebuilt binaries from third parties. Thankfully, making your own tap is as simple as creating a repository named "homebrew-tap" under your GitHub org or user, and getting a GitHub API token to push to it. cargo-dist will manage the contents of the repo for you.
+
+
+## Quickstart Part 1: Creating A Custom Tap
```

Comment:
nit: do we want a quickstart section with a part 1 and a part 2 instead of repeating the word quickstart?